app/templates/components/forms/user-profile-form.hbs

```diff
@@ -48,7 +48,7 @@
           <UiPopup
             @tagName="i"
             @class="info circle icon"
-            @content={{'Public profile name, if set, is displayed to other users on profiles and video sessions instead of real name.'}} />
+            @content={{'.. instead of a fantasy name.'}} />
```

Comment:
I think we just need to replace "real" with "fantasy" keeping the sentence intact. :thinking:

Reply:
I think this too 😄